common4j/src/main/com/microsoft/identity/common/java/telemetry/TelemetryEventStrings.java

```diff
@@ -28,6 +28,7 @@ public class TelemetryEventStrings {
     public static final class App {
         public static final String BUILD = EVENT_PREFIX + "application_build";
```

Comment:
Let's remove this EVENT_PREFIX, if it is not must have

Reply:
It would definitely need some checking in the code if anything is relying on these currently. This must only be done if can be safely done. If we cannot guarantee safety immediately, I suggest we do this in the next iteration.